crates/proto/src/xfer/dns_exchange.rs

```diff
@@ -196,10 +196,6 @@ where
                         Ok(()) => (),
                         Err(_) => {
                             warn!("failed to associate send_message response to the sender");
-
```

Comment:
Probably adding comments here to illustrate the situation could be better.

Reply:
Added, can you take a look and see if what I added makes sense?